### PR TITLE
'How Does Dogecoin Work' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Links need checking. Not sure how to generate the second link, which is to the whole 'Community and Ecosystem' **subsection** (not a specific question/article/stub).

### CODE REVIEW

File: content/dogepedia/articles/how-does-dogecoin-work.md [hunk 1]
```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
  
-A high-level overview of how Dogecoin works. ELI5 style.
\ No newline at end of file
+Magic and memes! (Just kidding. Kind of.) 
+
+First things first: Dogecoin *is* money.  
+
+Like other popular cryptocurrencies, Dogecoin works by utilising [blockchain technology](https://dogecoin.com/dogepedia/articles/what-is-a-blockchain). Unlike a lot of other popular cryptocurrencies, though, the fees to send Dogecoin are extremely small, and the speed by which Dogecoin transactions are confirmed is relatively fast.
```
Comment (MishaBoar) [inline content/dogepedia/articles/how-does-dogecoin-work.md:12]:
```suggestion
Like other popular cryptocurrencies, Dogecoin works by utilising [blockchain technology](/dogepedia/articles/what-is-a-blockchain). Unlike a lot of other popular cryptocurrencies, though, the fees to send Dogecoin are extremely small, and the speed by which Dogecoin transactions are confirmed is relatively fast.
```

Comment (MishaBoar) [reply content/dogepedia/articles/how-does-dogecoin-work.md:12]: Relative paths.

File: content/dogepedia/articles/how-does-dogecoin-work.md [hunk 2]
```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
...
\ No newline at end of file
+Magic and memes! (Just kidding. Kind of.) 
+
+First things first: Dogecoin *is* money.  
+
+Like other popular cryptocurrencies, Dogecoin works by utilising [blockchain technology](https://dogecoin.com/dogepedia/articles/what-is-a-blockchain). Unlike a lot of other popular cryptocurrencies, though, the fees to send Dogecoin are extremely small, and the speed by which Dogecoin transactions are confirmed is relatively fast. 
+
+Simply put, if you would like to send money to someone who is located anywhere in the world — or receive money from someone who is located anywhere in the world — *without* using a centralised bank and paying foreign currency exchange and transfer fees, then Dogecoin can work for you! 
+
+Furthermore, Dogecoin works by using it to purchase goods and services, and even for tipping people on popular social-media platforms like Reddit. See Community and Ecosystem [link to ‘Community and Ecosystem’ subsection] for more information on how Dogecoin can be used and how it works!
```
Comment (MishaBoar) [inline content/dogepedia/articles/how-does-dogecoin-work.md:16]:
```suggestion
Furthermore, Dogecoin works by using it to purchase goods and services, and even for tipping people on popular social-media platforms like Reddit. See [Community and Ecosystem](/dogepedia/#community-and-ecosystem) for more information on how Dogecoin can be used and how it works!
```

Comment (MishaBoar) [reply content/dogepedia/articles/how-does-dogecoin-work.md:16]: This is how to link. I am sending a separate PR to update the index file to include other in-page links.

**Review by MishaBoar — CHANGES_REQUESTED** [review]
This looks fine! I think in future iterations we can add a bit more details about PoW already in this introduction, but at this stage this is good as it is (and even long term it might be good to keep these entries fairly accessible and go more in-depth elsewhere!).

File: content/dogepedia/articles/how-does-dogecoin-work.md (hunk 1 above)
Comment (RichieJohn) [reply content/dogepedia/articles/how-does-dogecoin-work.md:12]: Great - thanks., MishaBoar!

File: content/dogepedia/articles/how-does-dogecoin-work.md (hunk 2 above)
Comment (RichieJohn) [reply content/dogepedia/articles/how-does-dogecoin-work.md:16]: Ah - gotcha! Thanks!

**Comment by RichieJohn** [comment]
This hasn't been merged yet. Owing to my newbie GitHub status, wondering whether I've done something incorrectly? Apologies if so! @tjstebbing @MishaBoar

**Review by MishaBoar — APPROVED** [review]
